### DISCUSSION TITLE
beginner

### DISCUSSION THREAD

**Discussion by Shailesh93602** [doocs/leetcode#958] [General] [answered]
hey i am a beginner here i have knowledge of cpp and I can provide codes in cpp

**Comment by yanglbme** [comment]
We welcome PRs :)

**Comment by Shailesh93602** [comment]
hey i am studying write now by providing solutions here can help me in future placements?

**Comment by Shailesh93602** [comment]
hey i want to add some cpp programs in which directory should add whether I add it in readme or direct in the desired programme?

**Comment by yanglbme** [comment, marked as answer]
hey, you can refer to other pull requests, such as #939 

- add your code in README.md & README_EN.md
- create a file Solution.cpp and add your code

**Comment by Shailesh93602** [comment]
i also have the knowledge of web development so can I contribute in development of front-end of the project?

**Reply by yanglbme** [reply]
yep, feel free to raise up any pull request :)

**Comment by Shailesh93602** [comment]
but i need to know that the web development process has started or is there any template on which should I start working
